src/components/Cart.js

```diff
@@ -11,6 +11,7 @@ const Cart = ({cart, removeFromCart}) => {
           </div>
         )}
       </div>
+      <button onClick={checkout} className="button button--add">Checkout</button>
```

Comment:
Given we're not actually _adding_ to the cart, what do you think about creating a new modifier called `button--checkout`?